both/collections/registration.js

```diff
@@ -32,7 +32,7 @@ Registrations.attachSchema({
       });
 
       if(uniqueMatch && (this.isInsert || docId != uniqueMatch._id)) {
-        return "notUnique";
+        return "notUnique"; // TODO this has no message?!?
```

Comment:
Looks like @FatBoyXPC added this in https://github.com/cubing/ccm/commit/7bfae507bcc0f279d58a7a472d2d0ad5bdacaac1. What's the behavior without a message? I'm guessing you just get a slightly hard to read message?

Lets fix this or make an issue to fix it.